api/attestationconfigapi/fetcher.go

```diff
@@ -18,9 +18,23 @@ import (
 
 const cosignPublicKey = constants.CosignPublicKeyReleases
 
+var (
+	// AzureSEVSNP is the Azure SEV-SNP variant.
+	AzureSEVSNP Variant = variant.AzureSEVSNP{}
+	// AWSSEVSNP is the AWS SEV-SNP variant.
+	AWSSEVSNP Variant = variant.AWSSEVSNP{}
+	// GCPSEVSNP is the GCP SEV-SNP variant.
+	GCPSEVSNP Variant = variant.GCPSEVSNP{}
+)
```

Comment:
Why is this needed?

Reply:
it is not needed, but since it's public a third party should be provided with valid defaults since `attestation/variant` is internal?